generator/types_generator.go

```diff
@@ -314,10 +314,23 @@ func (gen *goTypesGenerator) makeParameter(parameter *spec.Parameter) (*types.Ty
 			return enumType, nil
 		}
 
+	} else if parameter.Type == "file" {
+
+		return types.NewFile(strings.Title(parameter.Name), parameter.Required), nil
```

Comment:
suggestion: Use a wrapper strings.Title name of the wrapper could be xxx.MakePublic()

Reply:
Thanks for the suggestion but I don't think that it is a good practice to wrap std lib functionality which hasn't any further implications within the code where it is used.